action.yml

```diff
@@ -25,6 +25,13 @@ inputs:
   edr-command:
     description: The command to run, for example `edr monitor --slack-webhook <SECRET_SLACK_WEBHOOK>`.
     required: true
+  profile-target:
+    description: |
+      Which target to load for the given profile.
+      If specified, the target will be used for both the 'elementary' profile and your dbt project.
```

Comment:
Actually we can't determine what will be used for the `elementary` profile as it's passed in the `edr-command` argument, for instance - `edr-command: edr monitor -t ...`.

I think we can keep the description as _"Which target to load for the given profile."_

Reply:
On one hand, I agree. On the other, I think we should specify that it is only for dbt version check, because "real" target is passed along via `edr monitor --profile-target` 😕

Reply:
Got it, this makes sense.
Would you like to mention that in the description as well?